STRUCTURE.md

```diff
@@ -0,0 +1,59 @@
+# Structure
+Short description of project's structure.
+
+#### There are 7 packages which that are part of the project.
```

Comment:
There should probably be 7 packages described, shouldn't they?

Reply:
:+1: